electron/custom_install_steps.nsh

```diff
@@ -19,15 +19,15 @@
   File "${PROJECT_DIR}\electron\add_tap_device.bat"
 
   ${If} ${RunningX64}
-    ; nsExec::Exec hides the command prompt window.
-    nsExec::Exec 'add_tap_device.bat amd64'
+    nsExec::ExecToStack 'add_tap_device.bat amd64'
   ${Else}
-    nsExec::Exec 'add_tap_device.bat i386'
+    nsExec::ExecToStack 'add_tap_device.bat i386'
   ${EndIf}
 
   Pop $0
+  Pop $1

```

Comment:
Maybe add a comment about `nsExec::ExecToStack` pushing the return value and stdout to the stack.

Reply:
Done!